oaknational/Oak-Web-Application
# fix(a11y): aria hide icons and svgs

## Description

- adds `aria-hidden` to icons and svgs

## Issue(s)

fixes #785

## How to test

1. Go to {cloud link}
2. Check the things with a screen reader 
3. SR shouldn't announce svgs/icons which you don't care about


## Checklist
